client/my-sites/customer-home/cards/notices/site-resume-copy/index.js

```diff
@@ -0,0 +1,22 @@
+import { useTranslate } from 'i18n-calypso';
+import siteCopyIllustration from 'calypso/assets/images/customer-home/illustration--fireworks-v2.svg';
+import { NOTICE_SITE_RESUME_COPY } from 'calypso/my-sites/customer-home/cards/constants';
+import CelebrateNotice from 'calypso/my-sites/customer-home/cards/notices/celebrate-notice';
+
+const SiteResumeCopy = () => {
+	const translate = useTranslate();
+	return (
+		<CelebrateNotice
```

Comment:
Can we use the dark style similar to Atomic Revival, because we want them to take an action?

Reply:
Yes, actually, I needed to change the component from notice to task as this is the proper one.